apps/daimo-mobile/app.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
we moved `app.json` to `app.config.js` in this folder (to support multiple variants for local dev vs. prod app), could we move the delete this and move the `info.plist` (and any other?) change there?

also worth testing a fresh build on EAS when you do / if the permission pops up correctly on the dev build

Reply:
Fixed in https://github.com/daimo-eth/daimo/pull/305/commits/1c863d2e7dc371370f44a249c9c72d2d64deca1e